# Additional acknowledgements

### MRC CLIMB



## Methodology

Genomes were assembled using SPAdes (v.1.13) and contigs likely to be from plasmid origin were classified using mlplasmids (v1). The presence of AMR genes were determined using starAMR (v0.4.0) against the DTU Resfinder AMR gene database. Clustering of plasmid contigs carrying AMR genes were performed using R package kmer (v1.1.1) and custom python scripts were used to identify AMR gene combinations present on the same contig in each genome.

Viusaliations of AMR gene frequecies, distribution and plasmid sequence were carried out with ggplot2, iTol.embl.de and Easyfig.

### Figure 1

1. R, ggplot,
2. [mlplasmids](https://mgen.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000224#tab2)
3. [starAMR](https://github.com/phac-nml/staramr)
4. [Technical University of Denmark Resfinder DB](https://bitbucket.org/genomicepidemiology/resfinder/src/master/)

### Figure 2

1. [RAxML v8.2.9](https://sco.h-its.org/exelixis/web/software/raxml/index.html)
2. [iTol](http://itol.embl.de)
3. [hierBAPS](https://github.com/gtonkinhill/rhierbaps)

### Figure 3

1. [Treemer](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5930393/)

### Figure 4

1. [Easyfig](mjsull.github.io/Easyfi

### Figure 5

1. R, igraph



## Contact details

Twitter - [@bawee](https://twitter.com/bawee)


